src/markdown.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react/addons";
+import mdast from "mdast";
+import mdastReact from "mdast-react";
+
+import CodePane from "./code-pane";
+import Heading from "./heading";
+import Image from "./image";
+import Link from "./link";
+import List from "./list";
+import ListItem from "./list-item";
+import Quote from "./quote";
+import S from "./s";
+import Text from "./text";
+
+// We can't pass props into mdast-react directly, so we have to "bind" them
+// to spectacle components (ex. headings, strong/em/del)
+function spectacleComponent(component, boundProps = {}) {
+  return React.createClass({
+    render(){
+      const props = {...this.props, ...boundProps};
+      return React.createElement(component, {...props}, this.props.children);
+    }
+  });
+}
+
+// Would like to expose this config to the user
+const mdastConfig = {
```

Comment:
Could we just make this a property on `Markdown` to allow consumers to override the class if they want?